calico/etcddriver/driver.py

```diff
@@ -210,8 +211,9 @@ def _handle_init(self, msg):
         """
         # OK to dump the msg, it's a one-off.
         _log.info("Got init message from Felix %s", msg)
-        self._etcd_base_url = msg[MSG_KEY_ETCD_URL].rstrip("/")
+        self._etcd_base_url = msg[MSG_KEY_ETCD_URLS].pop().rstrip("/")
         self._etcd_url_parts = urlparse(self._etcd_base_url)
+        self._etcd_other_urls = msg[MSG_KEY_ETCD_URLS]
```

Comment:
Feels asymmetric not to `rstrip()` these too.  Haven't checked if that actually breaks anything.

I'd probably use a slice here on the principle that it's better to avoid modifying data that you're handed:
```
self._etcd_base_url = msg[MSG_KEY_ETCD_URLS][0]
self._etcd_other_urls = msg[MSG_KEY_ETCD_URLS][1:]
```